src/styles/index.css

```diff
@@ -0,0 +1,228 @@
+:root {
+  --accent-color: #00acdc;
+  --main-text-color: #616070;
+  --accent-text-color: #060b35;
+  --white: #fff;
+  --border-color: #f3f3f3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, serif;
```

Comment:
Fonts should be imported (using @font-face or a link to Google Fonts) if they are not system fonts. Ensure that 'Roboto' is either a system font or is imported correctly.